browser/updater/features.cc

```diff
@@ -3,19 +3,17 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-#include "brave/browser/mac_features.h"
+#include "brave/browser/updater/features.h"
 
 #include "build/build_config.h"
 
-static_assert(BUILDFLAG(IS_MAC), "Currently for macOS only");
-
-namespace brave {
+namespace brave::updater {
```

Comment:
nested namespaces are discouraged, but in particular you should not use `brave` as a top-level namespace just as upstream has removed all references to `chrome`. `brave_updater` would be ok

Reply:
I changed `brave::updater` to `brave_updater` now in 51029aeedc7c7a5f1b58b753dd0af1942b2edb84. The reason why I didn't pick `updater` is that `updater` is specific to Omaha 4. Our `brave_updater` may be Sparkle or Omaha 4 on macOS.